src/Extension/ValueResolver/ReportDirValueResolver.php

```diff
@@ -42,6 +42,10 @@ public function getValue(ContextInterface $context, array $settingValues, bool $
 
         $reportDir = $settingValues[static::REPORT_DIR_SETTING];
 
+        if (!is_dir($reportDir)) {
+            mkdir($reportDir, 0777, true);
```

Comment:
we never 0777, we always 0770 etc

Reply:
Ok. Where I can find this rule?
I also don't like `0777`, but I couldn't find any RFC and it is already implemented almost everywhere in sdk. It is kinda default permission to deal with shared files and directories.

Reply:
I see
Fair enough then